stories/components/section/section.scss

```diff
@@ -28,45 +28,45 @@
   background-color: var(--c-base);
 }
 
-.bix-section--primary {
-  background-color: var(--c-primary);
-  color: color("white");
-
+.bix-section--primary,
+.bix-section--primary-alt {
   .bix-input {
     background-color: transparent;
     color: inherit;
 
-    &:focus,
-    &:active {
-      background-color: color-mix(in srgb, var(--c-primary) 5%, black 30%);
-    }
-
     &::placeholder {
       color: rgba(white, 0.6);
     }
   }
 
-  legend {
+  legend,
+  .bix-button {
     color: inherit;
   }
+
+  .bix-button--unstyled {
+    &:hover,
+    &:focus {
+      color: inherit;
+    }
+  }
 }
 
-.bix-section--primary-alt {
-  background-color: var(--c-primary-alt);
+.bix-section--primary {
```

Comment:
@mejiaj something odd is going on here for the submit button. It's an inverse button, but the text and background are both white on hover now.
![image](https://github.com/user-attachments/assets/e8315895-8826-446f-bbb9-d2419f62e3fd)

Reply:
I've pushed up the changes to Drupal to allow contact form to be styled per the storybook component.

Reply:
@mattsqd is this issue still happening? Is that screenshot from the lando storybook example?

Reply:
Yes, I took it from my local storybook with the latest code.

Reply:
Okay, I was able to reproduce locally again. Must be a caching issue with Storybook.

Pushed up an update increasing inverse button specificity and should be good to go @mattsqd.